### ISSUE TITLE
can not add favorite currency list

### ISSUE THREAD

**Issue by simlegate** [jeppestaerk/alfred-currency-conversion#26]
It's correct to setup base currency.
<img width="577" alt="image" src="https://user-images.githubusercontent.com/2359726/165669065-c5ca0caa-4235-4dd4-8fa4-9cb0d22b27d4.png">
But it seem to can not recognize this command `curcon LIST`.
<img width="572" alt="image" src="https://user-images.githubusercontent.com/2359726/165669041-f199f5cc-7978-438f-b424-db61fb385a8e.png">
Is there some wrong operation for me?

Node: v17.9.0
Alfred: 4.6.4 Powerpack
alfred-currency-conversion: 0.2.0

**Comment by simlegate** [comment]
It seems that the new feature `Set your own favorite currency list` is not included in version `0.2.0`.
Is there deadline to release new version?

**Comment by MartinMalinda** [comment]
Yes, when I install this it seems like the code is quite different from master. I don't get the `favoriteCurrencies`.

I hover made this change in the source:

```js
const topCurrencies = process.env['CURRENCIES'] ?
  process.env['CURRENCIES'].split(',') :
  ["USD", "EUR", "JPY", "GBP", "CHF", "CAD", "AUD"];
const allCurrencies = ["AUD", "BGN", "BRL", "CAD", "CHF", "CNY", "CZK", "DKK", "EUR", "GBP", "HKD", "HRK", "HUF", "IDR", "ILS", "INR", "ISK", "JPY", "KRW", "MXN", "MYR", "NOK", "NZD", "PHP", "PLN", "RON", "RUB", "SEK", "SGD", "THB", "TRY", "USD", "ZAR"];
const currencies = [...new Set([...topCurrencies, ...allCurrencies])];
```

And I set comma separated currencies in environment variables and that works for me.
I guess I might lose this change on update but I don't mind that much.

**Comment by lidoror** [comment]
how can I paste it into the workflow?

**Comment by MartinMalinda** [comment]
![Screenshot 2022-07-26 at 9 31 03](https://user-images.githubusercontent.com/9468793/180949265-467a7c9e-55e2-4a8c-9d38-15f88851de1b.png)
![Screenshot 2022-07-26 at 9 31 46](https://user-images.githubusercontent.com/9468793/180949398-4faf99cf-4abf-4339-abc7-9a4a1ba5ed63.png)

There you can find the source. Good luck!

**Comment by lidoror** [comment]
did you replaced all three variable for it to work?

**Comment by MartinMalinda** [comment]
yes! I also set an environmenta variable


https://user-images.githubusercontent.com/9468793/180970315-a31fb71c-8e07-4b1a-9bdf-199705987c51.mov
